### PR TITLE
fix: add image component caption as children

### PR DESCRIPTION
**Pull request by jennspencer**
[![PR App][icn]][demo] | Fix RM-10049
:-------------------:|:----------:

## 🧰 Changes

cleaner way to add/update image caption

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: processor/transform/inject-components.ts [hunk 1]
```diff
@@ -18,8 +19,7 @@ const inject =
   };
 
 const injectComponents = (opts: Options) => (): Transform => tree => {
-  visit(tree, 'mdxJsxFlowElement', inject(opts));
-  visit(tree, 'mdxJsxTextElement', inject(opts));
+  visit(tree, isMDXElement, inject(opts));
```
Comment (kellyjosephprice) [inline processor/transform/inject-components.ts:22]: nice!

**Review by kellyjosephprice — CHANGES_REQUESTED** [review]
Code looks good, some nice cleanup.

Could you update the tests, and maybe add one add like `__tests__/transformers/image.test.ts` or something?

File: components/Image/index.tsx [hunk 2]
```diff
@@ -76,7 +79,7 @@ const Image = (Props: ImageProps) => {
               alt={alt}
               loading={lazy ? 'lazy' : 'eager'}
             />
-            <figcaption>{caption}</figcaption>
+            <figcaption>{children || caption}</figcaption>
```
Comment (kellyjosephprice) [inline components/Image/index.tsx:82]: I'm imaging a ticket in the future that gets immediately put in the backlog.

**Review by kellyjosephprice — APPROVED** [review]

File: components/Image/index.tsx (hunk 2 above)
Comment (jennspencer) [reply components/Image/index.tsx:82]: hahaha shhh

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.75.0-beta.64</b>](/readmeio/markdown/releases/tag/v6.75.0-beta.64)
<br>

**Pull request merged**